openid-sse-framework-1_0.txt

```diff
@@ -1884,19 +2206,9 @@ Tulshibagwale, et al.   Expires December 10, 2021              [Page 33]
       OPTIONAL.  Provides a short description of why the Transmitter has
       updated the status.
 
-   subject  OPTIONAL.  Specifies the Subject Principal for whom the
-      status has been updated.
-
-
-
-
-
-
-
-Tulshibagwale, et al.   Expires December 10, 2021              [Page 34]
-
-                          openid-sse-framework                 June 2021
-
+   subject
+      OPTIONAL.  Specifies the Subject Principal for whom the status has
```

Comment:
If this is optional, then we should specify that omitting the subject from the event means that the event applies to all subjects in the stream.